### ISSUE TITLE
How do you handle berman's ground-truth

### ISSUE THREAD

**Issue by AttackonMuggle** [honeygupta/UW-Net#11]
Thanks for your reply, but how did you prepare the ground-truth images. Since in evaluate_metrics.py line 41, the gt between 0-255 will all be converted to 0.

_Originally posted by @jinjidelinmouren in https://github.com/honeygupta/UW-Net/issues/10#issuecomment-1169921451_

**Comment by AttackonMuggle** [comment]
And you set the nan in gt to 0, did you do this for masking these points when calculating the error?

**Comment by honeygupta** [comment]
So that was a factor we added because of the pixel range of the depth maps in the Berman's dataset. AFAIR, the depth maps in the dataset are 16 bit images. Therefore, the code in line 41 converts the depth-map to 0-255 range.

**Comment by honeygupta** [comment]
> And you set the nan in gt to 0, did you do this for masking these points when calculating the error?

Yes, we only consider the non-nan pixels for computing the error and this is done to execute that.

**Comment by AttackonMuggle** [comment]
Get it. I also tried using the pretrained model to calculate the error, but couldn't get the results in Table 1. In addition, I also found that using the finetune model to get the depth, there will be white edges at the bottom of the depth map. Is this what affects my calculation of metrics?
![fakeB_depth_0_2](https://user-images.githubusercontent.com/69422383/177899911-a1f9057a-c326-4beb-b3e4-7aafd587823b.jpg)
I used test.py to get this.

**Comment by 9796l** [comment]
>hello I used internet images to test ,the file that i used is test.py to test.I used the pre-trained checkpoint,and i opened the result epoch_0.html ,i didn't find the predicted depth map.i didn't know the reason.when you test,how could you get the depth map?thank you very much.

![uw](https://user-images.githubusercontent.com/65001823/204962817-f83b54e0-8459-4065-b0d1-84312715f0be.png)

**Comment by 1980Ninja** [comment]
Hello Honey, I am having the same issue. Please can you point me out what I am doing wrong? I am not using RGBD images. Thanks! 
![fakeB_depth_0_57](https://github.com/honeygupta/UW-Net/assets/141886724/4bbdec1a-6040-4257-b114-f2edd2e13ea1)
![inputB_0_57](https://github.com/honeygupta/UW-Net/assets/141886724/cd1eab6e-8b90-4156-a70f-def7aae3b147)

**Comment by Zklb-250** [comment]
> Hello Honey, I am having the same issue. Please can you point me out what I am doing wrong? I am not using RGBD images. Thanks!
> ![fakeB_depth_0_57](https://private-user-images.githubusercontent.com/141886724/259589624-4bbdec1a-6040-4257-b114-f2edd2e13ea1.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDgzOTU0OTksIm5iZiI6MTcwODM5NTE5OSwicGF0aCI6Ii8xNDE4ODY3MjQvMjU5NTg5NjI0LTRiYmRlYzFhLTYwNDAtNDI1Ny1iMTE0LWYyZWRkMmUxM2VhMS5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIyMFQwMjEzMTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02YzE1YzNhNzE1YmUxNGU2ZDAwMjFlMTA5NmJhYzRhNDg1MGYzYmE0YTY4M2U4NGU3MGI5ZTIxNmI3ZDU2ZDVmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.CbRMwKELejJCy6cUFHH6U6tLJKmTA-rHDVPlq6-Tndo)
> ![inputB_0_57](https://private-user-images.githubusercontent.com/141886724/259589634-cd1eab6e-8b90-4156-a70f-def7aae3b147.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDgzOTU0OTksIm5iZiI6MTcwODM5NTE5OSwicGF0aCI6Ii8xNDE4ODY3MjQvMjU5NTg5NjM0LWNkMWVhYjZlLThiOTAtNDE1Ni1hNzBmLWRlZjdhYWUzYjE0Ny5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIyMFQwMjEzMTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04OTRmZmI2ZjE4MzlhNDI2MTFlZmVmM2U2NTNjNmFmZmQ1MTM0NzFkMTllZDJmYTY5N2Y5NTE2YWZjMzE4MzZiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.ilygZzhdcRmuyR9ITZBoKG2BDW0U0kf8mEMbG2HBIas)

Hi, I had the same problem with 3090 before. Then I switched to 2080 and got the right results. For older model frameworks, you might want to try an older graphics card.
